### ISSUE TITLE
Enhance a11y for `Avatar`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5404]
name: Avatar

### Description
12 accessibility violations were found in the `Avatar`.

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

#### [Components / Avatar / GroupMax](https://yamada-ui.github.io/yamada-ui/?path=/story/components-avatar--group-max)

Received:
[role="img"] elements must have alternative text (role-img-alt)

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/role-img-alt?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
